frontend/src/components/DataCard/SuggestedLabels.jsx

```diff
@@ -1,15 +1,21 @@
 import React from "react";
+import * as stringSimilarity from "string-similarity";
 
 export default function SuggestedLabels({ card, labels }) {
     if (labels.length <= 5) {
         return null;
     }
 
+    const labelStrings = labels.map(label => `${label.name} (${label.description})`);
+
+    let matches = stringSimilarity.findBestMatch(card.text.text, labelStrings);
+    matches = matches.ratings.sort((rating1, rating2) => parseFloat(rating2.rating) - parseFloat(rating1.rating)).slice(0, 5);
+
     return (
         <div className="suggestions">
             <h4>Suggested Labels</h4>
-            {card.text.similarityPair.slice(0, 5).map((opt, index) => (
```

Comment:
I think this method works really well. If it doesn't rely on any of the string similarity stuff on the backend, then we should remove all of that stuff in this MR.

Reply:
Yea! Ok, now that we like this way better, I'll get to removing all the different backend stuff we had previously.